source/system/String.ooc

```diff
@@ -26,6 +26,7 @@ String: class {
 		this _buffer free()
 		super()
 	}
+	safeFree: func { if (!this _isLiteral) this free() } // Will not free literals automatically
```

Comment:
Would this require much more than a private `_forceFree` kind of method, which gets called only by a static String cleanup for string literals ? So the code like `"a" free()` becomes a NOP for literals.